duckduckgo/chrome-zeroclickinfo
# Speed up grunt watch task

**Reviewer:**
@jdorweiler @russellholt 

## Description:
This has been an itch I've been meaning to scratch... Previous to this, our `grunt watch` task was building out our data files along with every change to the ui javascript, which was slowing me down a bit the last couple a weeks. I've broken the UI javascript off into a subdirectory, `js/ui/**` and the corresponding grunt browserify and watch tasks into `background` and `ui`.

## Steps to test this PR:
Pull down branch, run `npm run dev` or `grunt dev`. 
Make change to any file under `js/ui/*`. Notice the only files rebuilt are UI javascript.
Make change to `js/abp-preprocessed.js` and save. Notice only files rebuilt are background/data.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
